### PR TITLE
Fix misspelling errors

### PR DESCRIPTION
**Pull request by ka1bi4**
Fix some misspelling errors.

### CODE REVIEW

**Comment by ka1bi4** [comment]
Hi,
I fixed some misspelling errors.

File: docs/training_manual/basic_map/overview.rst [hunk 1]
```diff
@@ -64,7 +64,7 @@ all the layers stored in the same path of the project file under in the
 
 Moreover, you can set one or more folder as **Favorites**: search under your path
 and once you have found the folder, right click on it and click on ``Add as a
-Favorite``. You should then be able to see your folder in the |favourites|
+Favorite``. You should then be able to see your folder in the |favorites|
```
Comment (havatv) [inline docs/training_manual/basic_map/overview.rst:67]:
```suggestion
Favorite``. You should then be able to see your folder in the |favourites|
```
We can't change this only here. It is a reference (substitution)

File: docs/training_manual/basic_map/preparation.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ configured its installation.
 
 Let's get started right away!
 
-.. _tm_pepare_a_map:
+.. _tm_prepare_a_map:
```
Comment (havatv) [inline docs/training_manual/basic_map/preparation.rst:24]:
```suggestion
.. _tm_pepare_a_map:
```
This is an anchor, so all references to itmust also be changed

File: docs/training_manual/processing/john_snow.rst [hunk 3]
```diff
@@ -12,7 +12,7 @@ The dataset contains shapefiles with cholera deaths and pump locations, and an O
 
 .. figure:: img/john_snow/project.png
 
-The first thing to do is to calculting the Voronoi diagram (a.k.a. Thyessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
+The first thing to do is to calculating the Voronoi diagram (a.k.a. Thyessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
```
Comment (havatv) [inline docs/training_manual/processing/john_snow.rst:15]:
```suggestion
The first thing to do is to calculating the Voronoi diagram (a.k.a. Thiessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
```

**Review by havatv — CHANGES_REQUESTED** [review]
Thanks @ka1bi4 .
This looks good, but there are  a few things that have to be fixed before we can merge. See my suggestions and comments.

File: docs/training_manual/basic_map/preparation.rst (hunk 2 above)
Comment (DelazJ) [reply docs/training_manual/basic_map/preparation.rst:24]: better fix the spelling. Done!

File: docs/training_manual/processing/john_snow.rst (hunk 3 above)
Comment (DelazJ) [reply docs/training_manual/processing/john_snow.rst:15]: should this not be calculate?

File: docs/training_manual/processing/john_snow.rst [hunk 4]
```diff
@@ -12,7 +12,7 @@ The dataset contains shapefiles with cholera deaths and pump locations, and an O
 
 .. figure:: img/john_snow/project.png
 
-The first thing to do is to calculting the Voronoi diagram (a.k.a. Thyessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
+The first thing to do is to calculate the Voronoi diagram (a.k.a. Thiessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
```
Comment (havatv) [inline docs/training_manual/processing/john_snow.rst:15]:
```suggestion
The first thing to do is to calculate the Voronoi diagram (a.k.a. Thiessen
polygons) of the pumps layer, to get the influence zone of each pump.
The *Voronoi Diagram* algorithm can be used for that.
```
Too long line...

File: docs/training_manual/basic_map/overview.rst (hunk 1 above)
Comment (ka1bi4) [reply docs/training_manual/basic_map/overview.rst:67]: Ok, just notice you about this moment.

Comment (havatv) [reply docs/training_manual/basic_map/overview.rst]: @ka1bi4, I suggest we undo this change and merge. You can fix the substitution in a later PR.

File: docs/training_manual/processing/john_snow.rst (hunk 4 above)
Comment (ka1bi4) [reply docs/training_manual/processing/john_snow.rst]: Hm, maybe:
 To get the influence zone of each pump, calculate the the Voronoi diagram (a.k.a. Thiessen
polygons). The *Voronoi Diagram* algorithm can be used for that.

**Comment by ka1bi4** [comment]
Hi @Delaz and @havatv,
what changes still need to be made to perform the merge?

**Comment by DelazJ** [comment]
Thanks @ka1bi4  and sorry for the late merge

**Comment by ka1bi4** [comment]
@DelazJ, thanks. 
I thought I was missing something to close it 😊

**Pull request merged**